rd_ui/app/views/visualizations/map.html

```diff
@@ -0,0 +1 @@
+<div style='margin:20px;height:500px;width:500px;'></div>
```

Comment:
Can it inherit width from the containing element? So the user can control the width like he can for the other widgets.

As for height, maybe change this to be a visualization property (with some default value)?